Sources/ggwave/ggwave.cpp

```diff
@@ -17,9 +17,11 @@
 
 #ifdef ARDUINO
 #define ggprintf(...)
-#else
+#elseif DEBUG
 #define ggprintf(...) \
     g_fptr && fprintf(g_fptr, __VA_ARGS__)
+#else
+#define ggprintf(...)
 #endif
```

Comment:
I would rather have something like:

```c++
#ifdef GGWAVE_DISABLE_LOG
#define ggprintf(...)
#else
#ifdef ARDUINO
#define ggprintf(...)
#else
#define ggprintf(...) \
    g_fptr && fprintf(g_fptr, __VA_ARGS__)
#endif
#endif
```

If you agree with this change, please open the PR directly to the [ggwave](https://github.com/ggerganov/ggwave) repo and I will propagate it from there to this repo

Reply:
Sure thing, the proposed approach works well too...